candidate/models.py

```diff
@@ -0,0 +1,93 @@
+from __future__ import unicode_literals
+from django.db import models
+from django.utils.encoding import python_2_unicode_compatible
+
+from ballot_measure.models import BallotMeasureChoice
+
+
+class SocialMediaModel(models.Model):
+    class Meta:
+        abstract = True
+
+    photo_url = models.ImageField(blank=True)
+    website_url = models.URLField(
+        help_text='URL for the official website.', blank=True)
+    facebook_url = models.URLField(
+        help_text='URL for the Facebook page.', blank=True)
+    twitter_url = models.URLField(
+        help_text='URL for the Twitter page.', blank=True)
+
+
+@python_2_unicode_compatible
+class Party(SocialMediaModel):
+    """
+    """
+    name = models.CharField(
+        max_length=255, help_text='The party name.')
+
+    def __str__(self):
+        return "The %s party" % self.name
+
+
+@python_2_unicode_compatible
+class Person(SocialMediaModel):
+    """
+    """
+    first_name = models.CharField(
+        max_length=255, help_text='The candidate\'s first name.')
+    middle_name = models.CharField(
+        max_length=255, blank=True, null=True,
+        help_text='The candidate\'s middle name.')
+    last_name = models.CharField(
+        max_length=255, blank=True, null=True,
+        help_text='The candidate\'s last name.')
+    party = models.ForeignKey('Party', blank=True, null=True)
+
+    def __str__(self):
+        return "%s, %s%s" % (
+            self.last_name, self.first_name,
+            '' if self.middle_name is None else ' %s.' % self.middle_name[0])
+
+
+@python_2_unicode_compatible
+class Office(models.Model):
+    """
+    """
+    name = models.CharField(
+        max_length=255, help_text='The office name.')
+    description = models.CharField(
+        max_length=1024, help_text='The office description.')
+    locality = models.ForeignKey('locality.Locality')
+
+    def __str__(self):
+        return "Office for %s in %s" % (
+            self.name, self.locality.dereference())
+
+
+@python_2_unicode_compatible
+class Election(models.Model):
```

Comment:
> I was trying to use the word Election to be the event when citizens vote.

I think `Election` is used both for single office (that's the main Wikipedia entry) and the overall event. I'm not clear why you prefer to keep `Election` for the event, rather than the single office contest. From the reading I did online, I felt the stronger meaning was for the single office contest.

These vocabulary will be used throughout the app, including the front-end, and keeping things consistent across all levels will make things run more smoothly. Perhaps we can get advice from others for these terms?

As for `Office` vs. `Election` (as I'm using it here), I would prefer to keep them separate. `Office` exists independent of election or not. It's like a generic government object. I've used that same type of abstraction in other places, for example `Person` vs. `IndividualBenefactor`, etc. and I think separating static metadata (office, person) from functional metadata (election, finance) is a nice, consistent way to keep our models focused and as light as possible.